numpy/core/src/multiarray/compiled_base.c

```diff
@@ -299,7 +304,14 @@ arr_histogramdd_uniform(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kw
             min[d] = e[0];
             max[d] = e[bins[d]];
 
-            if (bins[d] <= 0 || (max[d] - min[d]) <= 0)
+
+            if (bins <= 0)
```

Comment:
Lost a `[d]`

Reply:
Turns out the public interface will never get there so I didn't add a unittest for this. The check is potentially valuable in the future so I'd leave it in. Is that OK?